__testfixtures__/integration/all.output.js

```diff
@@ -25,8 +25,8 @@ test('it renders again', function(assert) {
   this.render(hbs`{{foo-bar}}`);
 
   let selector = '.foo input';
-  assert.equal(findAll(selector).length, 1);
-  assert.equal(find(selector).value, 'foo');
+  assert.equal(findAll(this.$(selector)[0]).length, 1);
```

Comment:
I'm not sure if this is correct. We should discuss, as it feels weird to pass a single element to `findAll`.

Reply:
Good point, I agree.  We should not add `[0]` to `findAll`, I think we should use `this.$(selector).get()` here.  `.get()` returns a normal array of native DOM elements matching the jQuery selector.

Docs here: http://api.jquery.com/get/